scene/3d/camera.cpp

```diff
@@ -99,9 +99,15 @@ void Camera::_notification(int p_what) {
 
 		case NOTIFICATION_ENTER_WORLD: {
 
-			bool first_camera = get_viewport()->_camera_add(this);
-			if (!get_tree()->is_node_being_edited(this) && (current || first_camera))
-				make_current();
+			// Needs to track the Viewport  because it's needed on NOTIFICATION_EXIT_WORLD
+			// and Spatial will handle it first, including clearing its reference to the Viewport,
```

Comment:
Should we not continue supporting the is_node_being_edited() check?

Reply:
Actually, that was the greatest point of uncertainty for me.
I think that, when the camera enters the world, it's not really needed to do that check, at least after my changes.
It's still done on exit world because there I found a reason for it.
To sum it up, please give me some insight about this.

Reply:
Specifically, I don't want to break the intent of this commit: https://github.com/godotengine/godot/commit/2db78e8f87fc208711bf8625fcb679723ce46fb7
If you can explain its purpose, I will make sure this PR doesn't regress it.

Reply:
The idea is that users may not be setting a camera to current, but if one exists, it should be used anyway.
This removes the confusion that many users had when testing this stuff.

Reply:
So the code above should be alright, no need to delete it.